Library/Homebrew/dev-cmd/pr-publish.rb

```diff
@@ -21,7 +21,7 @@ def pr_publish_args
   def pr_publish
     pr_publish_args.parse
 
-    odie "You need to specify a pull request number!" if Homebrew.args.named.empty?
+    odie "You need to specify at least one pull request number!" if Homebrew.args.named.empty?
```

Comment:
@jonchang There's a DSL to do this so you don't need the `odie`: https://github.com/Homebrew/brew/pull/7202/files#r396070600

Reply:
I tested it locally and it didn’t seem to work with multiple arguments. Is there another DSL for that?

Reply:
`min_named`. Sorry, thought from the help text it only took a single arg.